app/controllers/merchant/items_controller.rb

```diff
@@ -1,5 +1,67 @@
 class Merchant::ItemsController < Merchant::BaseController
+
   def index
     @merchant = current_user.merchant
   end
+
+  def new
+    @merchant = current_user.merchant
+    @item = Item.new
+  end
+
+  def create
+    @merchant = current_user.merchant
+    @item = @merchant.items.create(item_params)
+    @item.save ? flash_and_redirect(@item) : sad_path(@item, "new")
+  end
+
+  def edit
+    @item = current_user.merchant.items.find(params[:id])
+  end
+
+  def update
+    @item = Item.find(params[:id])
+    params.include?(:active?) ? update_status(@item) : update_item(@item)
```

Comment:
I feel this can be combined. I will play when I pull it down and see what I can do.